machine_configuration/index.adoc

```diff
@@ -44,6 +44,11 @@ include::modules/machine-config-drift-detection.adoc[leveloffset=+1]
 include::modules/checking-mco-status.adoc[leveloffset=+1]
 include::modules/checking-mco-node-status.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about feature gates, see xref:../nodes/clusters/nodes-cluster-enabling-features.adoc[Enabling feature sets using the web console].
```

Comment:
```suggestion
* For more information about feature gates, see xref:../nodes/clusters/nodes-cluster-enabling-features.adoc#nodes-cluster-enabling-features-console_nodes-cluster-enabling[Enabling feature sets using the web console].
```

This link should point directly to the target module.

Reply:
Outside this PR, but the target module title should read: `Enabling feature sets by using the web console`.

![Screenshot from 2024-07-30 12-37-00](https://github.com/user-attachments/assets/e48f6c86-bc1f-4254-a3e1-9376add811ad)